bin/delete_holdings_by_uuid.rb

```diff
@@ -19,11 +18,11 @@ def main
   org_counts = {}
 
   ARGV.each do |path|
-    waypoint = Services.progress_tracker.new(1000)
+    marker = Services.progress_tracker.new(1000)
     inf = File.open(path, "r")
     inf.each_line do |line|
-      waypoint.incr
-      waypoint.on_batch do |wp|
+      marker.incr
+      marker.on_batch do |wp|
```

Comment:
maybe we don't want to call the variable wp anymore

Reply:
Good catch.